### ISSUE TITLE
Mounting with TLS failes with FileNotFoundError: [Errno 2] No such file or directory: '/sbin/status'

### ISSUE THREAD

**Issue by yonatankarimish** [aws/efs-utils#163]
**TL;DR** - Mounting an EFS file system on my dev laptop is successful, but trying to mounts with the **tls** flag fails.

Hi
Following your installation guides [here](https://docs.aws.amazon.com/efs/latest/ug/using-amazon-efs-utils.html), I installed the amazon efs utils tools along with botocore and the upgraded version of stunnel.

Using the following parameters, the efs mount command is successful:
**file system id:** fs-0929f82d6518ac6dd
**mount folder:** /mnt/norman/
**command:** sudo mount -t efs fs-0929f82d6518ac6dd /mnt/norman/

However, adding the tls flag fails to mount the file system:
**command:** sudo mount -t efs **-o tls** fs-0929f82d6518ac6dd /mnt/norman/
```
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.14) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Traceback (most recent call last):
  File "/sbin/mount.efs", line 3774, in <module>
    main()
  File "/sbin/mount.efs", line 3752, in main
    mount_tls(
  File "/sbin/mount.efs", line 2883, in mount_tls
    with bootstrap_tls(
  File "/usr/lib/python3.8/contextlib.py", line 113, in __enter__
    return next(self.gen)
  File "/sbin/mount.efs", line 1589, in bootstrap_tls
    start_watchdog(init_system)
  File "/sbin/mount.efs", line 1432, in start_watchdog
    proc = subprocess.Popen(
  File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/sbin/status'
```

Checking the /sbin directory shows there is indeed no file named "status" there:
```
yonatan@Yonatan-Work-Laptop:/mnt$ ls /sbin | grep status
aa-status
apparmor_status
e2mmpstatus
```

The efs mount logs do not show any errors:
```
yonatan@Yonatan-Work-Laptop:/mnt$ cat /var/log/amazon/efs/mount.log
2023-03-02 02:18:08 IST - INFO - version=1.34.5 options={'rw': None, 'tls': None}
2023-03-02 02:18:08 IST - DEBUG - Identified init system: init
2023-03-02 02:18:08 IST - DEBUG - Not testing network on non-systemd init systems
2023-03-02 02:18:08 IST - DEBUG - Using dns_name_suffix amazonaws.com in config section [mount]
2023-03-02 02:18:08 IST - INFO - binding 20783
2023-03-02 02:18:08 IST - DEBUG - Removed /var/run/efs/efs-utils-lock successfully
2023-03-02 02:18:08 IST - DEBUG - Closing socket used to choose TLS port 20783.
```

Any help on this will be much appreciated.

**Comment by RyanStan** [comment]
Hi @yonatankarimish , can you provide the AMI/ Operating System that you are using?

**Comment by yonatankarimish** [comment]
@RyanStan Using Windows 11 on my local development laptop. 

The folder being used as the mount target resides within WSL
The commands were also executed through WSL.

(sorry for not providing this info earlier)

**Comment by arnavgup1** [comment]
Hi @yonatankarimish, could you provide more insight on what distribution your WSL is? Also, could you send the stunnel logs from when you attempt to mount with tls? You can enable the stunnel logs with these [instructions](https://github.com/aws/efs-utils#troubleshooting).

**Comment by arnavgup1** [comment]
@yonatankarimish stunnel writes its logs to `/var/log/amazon/efs`. The logs will be stored in a file called `mount-watchdog.log`.
